### DISCUSSION TITLE
Cross platform code editor recommendations?

### DISCUSSION THREAD

**Discussion by Wysardry** [davidgiven/ack#299] [General]
Does anyone have any recommendations for a cross-platform code editor to use with ACK?

I'm looking for something that will work with Windows 10, WSL2 (Debian), MSYS, Debian 13 and Raspberry Pi OS as I may need to switch between those.

Are there any editors that have syntax highlighting for any of the ACK-specific files?

**Comment by davidgiven** [comment]
Well... vim? [/wry]

I personally use a combination of VSCode (with the vim plugin) and vim itself, depending on what I'm doing. There isn't any syntax highlighting, unfortunately.
